antlr4ide.tests/src/main/java/com/github/jknack/antlr4ide/console/TestAppender.java

```diff
@@ -0,0 +1,79 @@
+package com.github.jknack.antlr4ide.console;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.log4j.AppenderSkeleton;
+import org.apache.log4j.Level;
+import org.apache.log4j.spi.LoggingEvent;
+
+class TestAppender extends AppenderSkeleton {
+	private final List<String> errors;
+	private final List<String> warnings;
+	private final List<String> infos;
+	private final List<String> debugs;
+	private final List<String> traces;
+	private final List<String> others;
```

Comment:
Yes, as I am using Level[] logLevels = Level.values(); in GeneralPreferencePage on line 34. FATAL is part of the default log levels. Of course it would be possible to filter some log-levels, but I am not fully convinced to implement this as I am unsure if users will ever change the default log-level.